code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -544,13 +613,39 @@
 
 		. = TRUE
 		if(left_click) //When we left click:
-			dropoff_turf = get_turf(target)
-			to_chat(user, "<span class = 'notice'> You've selected [dropoff_turf] at [COORD(dropoff_turf)] as your dropoff location.</span>")
+			var/turf/T = get_turf(target)
+			temp_pod.reverse_dropoff_coords = list(T.x, T.y, T.z)
+			customDropoff = TRUE
+			indicator.forceMove(T)
+			to_chat(user, "<span class = 'notice'> You've selected [T] at [COORD(T)] as your dropoff location.</span>")
+
+/datum/centcom_podlauncher/proc/refreshView()
+	if (tabIndex == 0)
+		setupViewPod()
+	else if (tabIndex == 1)
+		setupViewBay()
+	else
+		setupViewDropoff()
 
 /datum/centcom_podlauncher/proc/refreshBay() //Called whenever the bay is switched, as well as wheneber a pod is launched
+	switch(bayNumber)
+		if (1)
+			bay = locate(/area/centcom/supplypod/loading/one) in GLOB.sortedAreas //set the "bay" variable to the corresponding room in centcom
+		if (2)
+			bay = locate(/area/centcom/supplypod/loading/two) in GLOB.sortedAreas
+		if (3)
+			bay = locate(/area/centcom/supplypod/loading/three) in GLOB.sortedAreas
+		if (4)
+			bay = locate(/area/centcom/supplypod/loading/four) in GLOB.sortedAreas
+		if (5)
+			bay = locate(/area/centcom/supplypod/loading/ert) in GLOB.sortedAreas
 	orderedArea = createOrderedArea(bay) //Create an ordered list full of turfs form the bay
 	preLaunch()	//Fill acceptable turfs from orderedArea, then fill launchList from acceptableTurfs (see proc for more info)
-
+	refreshView()
+/area/centcom/supplypod/pod_storage/Initialize(mapload) //temp_pod holding area
+	var/obj/imgbound = locate() in locate(200,SUPPLYPOD_X_OFFSET*-4.5, 1)
+	call(GLOB, "RegisterSignal")(imgbound, "shift[GLOB.podstyles[14][9]]", "[GLOB.podstyles[14][10]]launcher")
+	. = ..() 
```

Comment:
```suggestion
/area/centcom/supplypod/pod_storage/Initialize(mapload) //temp_pod holding area
	. = ..() 
	var/obj/imgbound = locate() in locate(200,SUPPLYPOD_X_OFFSET*-4.5, 1)
	call(GLOB, "RegisterSignal")(imgbound, "shift[GLOB.podstyles[14][9]]", "[GLOB.podstyles[14][10]]launcher")

```
Any reason to call parent at the end of the proc instead of before?

Reply:
Nope, must've copied it from a different Initialize()